ui-charts/src/spaceTimeChart/components/PathLayer.tsx

```diff
@@ -336,8 +345,48 @@ export const PathLayer = ({
     [path]
   );
 
+  const drawBorder = useCallback<DrawingFunction>(
+    (ctx, stcContext) => {
+      if (!border) return;
+      const borderWidth = border.width || 1;
+      const mainPathStyle = STYLES[level];
+      const totalPathWidth = border.offset * 2 + mainPathStyle.width;
+
+      const segments = getPathSegments(stcContext);
+      ctx.save();
+      const drawSegments = (lineWidth: number, borderColor = border.color) => {
+        ctx.strokeStyle = borderColor;
+        ctx.lineWidth = lineWidth;
+        ctx.beginPath();
+        segments.forEach(({ x, y }, i) => {
+          if (x === segments[i - 1]?.x && y === segments[i - 1]?.y) return;
+          if (i === 0) {
+            ctx.moveTo(x, y);
+          } else if (i === segments.length - 1) {
+            ctx.lineTo(x - border.offset / 2, y);
+          } else {
+            ctx.lineTo(x, y);
+          }
+        });
+        ctx.stroke();
+      };
+
+      drawSegments(totalPathWidth + borderWidth * 2);
+
+      if (border.backgroundColor) {
+        drawSegments(totalPathWidth, border.backgroundColor);
```

Comment:
This is a bit confusing: when there is no background color, I would've expected a thin `width`-sized border `offset` away from the path. However a thick path is painted instead:

![out](https://github.com/user-attachments/assets/1a1ac3a5-699a-4d33-a9c0-957a230efbdc)

Reply:
(I suppose the right way to fix this would be to leave the space between the path and the border transparent, which would be tricky to do? Perhaps a workaround would be to use white.)

Reply:
Sorry i forgot to answer, but i fix this using globalCompositeOperation